src/lib.rs

```diff
@@ -1390,6 +1390,25 @@ pub fn __private_api_log(
     );
 }
 
+// WARNING: this is not part of the crate's public API and is subject to change at any time
+#[doc(hidden)]
+pub fn __private_api_log_lit(
```

Comment:
Is `lit` a type-o or is `lite` too long?

Reply:
It's short for `literal`, not `lite`.